lab-maddy/__test__/sll.test.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const SLL = require('../lib/sll');
+require('jest');
+
+describe('Testing Singly Linked List', function(){
+  describe('default properties', ()=>{
+    beforeAll(done => {
+      this.sll = new SLL();
+      done();
+    });
+    test('should have a head property of null', ()=> {
+      expect(this.sll).toHaveProperty('head');
+      expect(this.sll.head).toBeNull();
+    });
+  });
+  describe('');
+});
+
+describe('#reverse', () => { //still in the process of writing this one!!!
+    beforeAll(done => {
+      // this.sll = new SLL()
+      done()
+    })
+    describe('reverse the pointers in the linked list', () => {
+      test('should add a new node to Head', () => {
+        this.sll.prepend(2)
+        expect(this.sll.head.val).toBe(2)
+        expect(this.sll.head.next).toBeNull()
+      })
+      test('should add a second node to the list', () => {
+        this.sll.prepend(3)
+        expect(this.sll.head.val).toBe(3)
+        expect(this.sll.head.next.val).toBe(2)
+        expect(this.sll.head.next.next).toBeNull()
+      })
+    })
+  })
```

Comment:
Remove this line. It is extra and closing the describe block causing all sorts of errors in your test file. You will probably get a bunch of indentation error once you remove it.